src/mantine-hooks/src/use-idle/use-idle.ts

```diff
@@ -0,0 +1,30 @@
+import { useState, useEffect, useRef } from 'react';
+
+type eventsType = MouseEvent | KeyboardEvent | TouchEvent | Event;
+
+export function useIdle(timeout: number, cb?: (e?: eventsType) => void) {
```

Comment:
Remove second argument `cb`, it brings no value, user can subscribe to those events if needed in other part of application